cli/nep17_test.go

```diff
@@ -83,6 +83,11 @@ func TestNEP17Balance(t *testing.T) {
 				e.checkNextLine(t, "^\\s*Updated\\s*:\\s*"+strconv.FormatUint(uint64(index), 10))
 			}
 		}
+
+		e.checkNextLine(t, "^\\s*$")
+		addr4, err := address.StringToUint160("NbxpLNCCSWZ9BkYpCYT8NfN1uoxq9Rfbrn") // deployed verify.go contract
+		require.NoError(t, err)
+		e.checkNextLine(t, "^Account "+address.Uint160ToString(addr4))
```

Comment:
Unrelated?

Reply:
Why? TestNEP17Balance will fail without it (because I deployed a new contract to the chain to test the changes).